src/components/Button.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function Button({ name, color, wide }) {
+  const buttonStyle = {
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    width: '25%',
+    backgroundColor: '#ffa502',
+    margin: 4,
+    fontSize: 20,
+    fontWeight: 'bold',
+    border: '2px solid #000',
+  };
+
+  if (wide) {
+    buttonStyle.width = '50%';
+  }
+  if (color) {
+    buttonStyle.backgroundColor = color;
+  }
```

Comment:
- The default color should be `orange` as mentioned on Pathwright, see [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/colorProps.png)

**[hint]**
You can set the default value of a prop by using `defaultProps`as explained [here](https://reactjs.org/docs/typechecking-with-proptypes.html#default-prop-values)

Reply:
fixed